frontend/packages/ux-editor/src/containers/FormItemContext.test.tsx

```diff
@@ -110,30 +88,25 @@ describe('FormItemContext', () => {
     await user.click(button);
 
     await waitFor(async () =>
-      expect((await screen.findByTestId('formItemId')).textContent).toEqual(mockFormItem.id),
+      expect((await screen.findByTestId('formItemId')).textContent).toEqual(component1Mock.id),
     );
     await waitFor(async () =>
-      expect((await screen.findByTestId('formItem.id')).textContent).toEqual(mockFormItem.id),
+      expect((await screen.findByTestId('formItem.id')).textContent).toEqual(component1Mock.id),
     );
     await waitFor(async () =>
       expect((await screen.findByTestId('formItem.itemType')).textContent).toEqual(
-        mockFormItem.itemType,
+        component1Mock.itemType,
       ),
     );
   });
 
   it('should render id and itemType when calling handleEdit with truthy updatedForm', async () => {
     const user = userEvent.setup();
-    const mockFormItem: FormContainer = {
-      id: 'id',
-      itemType: 'CONTAINER',
-      type: ComponentType.Group,
-    };
     render(() => {
       const { formItemId, formItem, handleEdit } = React.useContext(FormItemContext);
       return (
         <>
-          <button data-testid='button' onClick={() => handleEdit(mockFormItem)} />
+          <button data-testid='button' onClick={() => handleEdit(component1Mock)} />
```

Comment:
It seems like this is used several places in the test, is it possible to refactor it to a component? 😄

Reply:
Done